src/openforms/setup.py

```diff
@@ -113,6 +115,57 @@ def get_success_url(self):
     CookieGroupBaseProcessView.get_success_url = get_success_url
 
 
+def monkeypatch_mozilla_django_oidc_get_from_settings():
+    def get_next_url(request, redirect_field_name):
```

Comment:
this should be based on our CORS settings and own utility for safe_url check, see https://open-forms.readthedocs.io/en/latest/developers/backend/core/utils.html#openforms.utils.redirect.allow_redirect_url